src/main/java/fr/ospiea/oscra/restsecurity/RESTAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,22 @@
+package fr.ospiea.oscra.restsecurity;
+
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by taozheng on 27/09/2016.
+ */
+@Component
+public class RESTAuthenticationEntryPoint implements AuthenticationEntryPoint {
+    @Override
+    public void commence(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, AuthenticationException e) throws IOException, ServletException {
```

Comment:
I'm not expert about security. but I feel all these classes just to implement a form secured system is overkilled. Are you sure that there isn't simplier method?